schemas/user/createUserSchema.js

```diff
@@ -17,7 +23,7 @@ module.exports = {
       },
     },
   },
-  firstNameNull: {
+  firstName: {
     custom: {
```

Comment:
change the custom field for  firstName: {
    notEmpty:{errorMessage:"firstName is null"}